app/include_upgrade_testnet.go

```diff
@@ -0,0 +1,21 @@
+//go:build testnet
+
+package app
+
+import (
+	"github.com/babylonlabs-io/babylon/app/upgrades"
+	v1 "github.com/babylonlabs-io/babylon/app/upgrades/v1"
+	"github.com/babylonlabs-io/babylon/app/upgrades/v1/testnet"
+)
+
+// init is used to include v1 upgrade testnet data
+// it is also used for e2e testing
+func init() {
+	Upgrades = []upgrades.Upgrade{v1.CreateUpgrade(v1.UpgradeDataString{
```

Comment:
only place beside tests that imports testnet and is included only if there is a `testnet` in build tags